payments/swish/direct.md

```diff
@@ -0,0 +1,271 @@
+  a [CallbackURL][callback-url] in the `POST` request. If `callbackURL` is set,
+  Swedbank Pay will send a postback request to this URL when the consumer has
+  fulfilled the payment.
+
+## Purchase flow
+
+The sequence diagram below shows the three requests you have to send to
+Swedbank Pay to make a purchase.
+
+```mermaid
+sequenceDiagram
+  activate Browser
+  Browser->>-Merchant: start purchase
+  activate Merchant
+  Merchant->>-SwedbankPay: POST <Swish payment> (operation=PURCHASE)
+  activate  SwedbankPay
+  note left of Merchant: First API request
+   SwedbankPay-->>-Merchant: payment resource
+   activate Merchant
+
+  Merchant-->>- SwedbankPay: POST <Sales Transaction> (operation=create-sale)
+  activate  SwedbankPay
+   SwedbankPay-->>-Merchant: sales resource
+  activate Merchant
+  note left of Merchant: POST containing MSISDN
+  Merchant--x-Browser: Tell consumer to open Swish app
+  Swish_API->>Swish_App: Ask for payment confirmation
+  activate Swish_App
+  Swish_App-->>-Swish_API: Consumer confirms payment
+  activate Swish_API
+
+  Swish_API-->>- SwedbankPay: Payment status
+  activate  SwedbankPay
+   SwedbankPay-->>-Swish_API: Callback response
+  activate Swish_API
+  Swish_API->>-Swish_App: Start redirect
+  activate Swish_App
+  
+  Swish_App--x-Browser: Redirect
+  activate Merchant
+  Merchant->>- SwedbankPay: GET <Sales transaction>
+  activate  SwedbankPay
+   SwedbankPay-->>-Merchant: Payment response
+  activate Merchant
+  Merchant-->>-Browser: Payment Status
+```
+
+{
+  "payment": {
+    "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c",
+    "number": 1234567890,
+    "created": "2016-09-14T13:21:29.3182115Z",
+    "updated": "2016-09-14T13:21:57.6627579Z",
+    "instrument": "swish",
+    "operation": "Purchase",
+    "intent": "Sale",
+    "state": "Ready",
+    "currency": "NOK",
+    "amount": 1500,
+    "remainingReversalAmount": 1500,
+    "description": "Test Purchase",
+    "payerReference": "AB1234",
+    "initiatingSystemUserAgent": "PostmanRuntime/3.0.1",
+    "userAgent": "Mozilla/5.0...",
+    "language": "nb-NO",
+    "prices": { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/prices" },
+    "transactions": { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/transactions" },
+    "reversals": { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/reversals" },
+    "sales": { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/sales" },
+    "urls" : { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/urls" },
+    "payeeInfo" : { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/payeeInfo" },
+    "settings": { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/settings" }
+  },
+  "operations": [
+    {
+      "href": "http://localhost:5001/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c",
+      "rel": "update-payment-abort",
+      "method": "PATCH"
+    }
+  ]
+}
+
+
+{
```

Comment:
These operation code examples seem a bit astray. Shouldn't they at least be wrapped in code fences and have a code header?